tests/test_api.py

```diff
@@ -390,6 +397,293 @@ def test_operations(self):
         self.assertEqual(op1_args['description'], 'Example desc.')
 
 
+class TestAddProject(TestCase):
+    """Unit tests for add_project"""
+
+    def setUp(self):
+        """Load initial values"""
+
+        self.user = get_user_model().objects.create(username='test')
+        self.ctx = BestiaryContext(self.user)
+
+        self.eco = Ecosystem.objects.create(name='Eco-example')
+
+    def test_add_new_project(self):
+        """Check if everything goes OK when adding a new project"""
+
+        proj = api.add_project(self.ctx,
+                               self.eco.id,
+                               name='example-name',
```

Comment:
As `name` is a required parameter, it's more clear if you pass it just its value (`example-name`) and not with as a keyword argument. Keywords usually imply they are optional.
Please fix it everywhere else.